test/conformance/chainsaw/policy-validation/policy/any-pattern-context /chainsaw-test.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  creationTimestamp: null
+  name: any-pattern-context
+spec:
+  steps:
+  - name: step-01
+    try: 
+    - apply:
+        file: policy.yaml
+  - name: step-03
+    try:
+    - apply:
+        expect:
+        - check:
+            ($error != null): true 
+        file: pod.yaml
```

Comment:
We need to verify the output message resulting from applying the policy to the pod. I recommend doing something like this:
```
  - name: step-03
    try:
    - script:
        content: kubectl apply -f pod.yaml
        check:
          # This check ensures the contents of stderr are exactly as shown.
          ($stderr): |-
            "The message goes here"
```